CHANGELOG.md

```diff
@@ -8,32 +8,86 @@ core section.***
 - ***When possible please provide the number of the pull request(s) containing the 
 changes in the following format: PR #1234***
 
-### LORIS 23.0 (Released: ??)
```

Comment:
Why are you deleting the release date? We should keep it and fill it in right before releasing.. that's one of the most important parts of change logs.

Reply:
oh I figured we would have to fill it post release which defeats the purpose cause the date would just stay as `??`

I'll readd it

Reply:
Even if it was filled post-release, when viewing other software's change logs I'm often looking for "how outdated is version xxx?" for some old version, the most recent date of the most recent one isn't super important..